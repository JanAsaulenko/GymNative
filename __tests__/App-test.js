/**
 * @format
 */

import("react-native");
import React from "react";
import { shallow } from "enzyme"
const { Form } = require('../Components/Form');



describe('rendering', () =>
{
  it('create snapshot', () =>
  {
    const component = shallow(<Form />);
    expect(component).toMatchSnapshot();
  });
  it('equals', () =>
  {
    expect(2 + 3).toBe(5)
  });

});
