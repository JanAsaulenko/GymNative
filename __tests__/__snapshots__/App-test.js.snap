// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering create snapshot 1`] = `ShallowWrapper {}`;
